zebra-state/src/request.rs

```diff
@@ -6,6 +6,7 @@ use std::{
     sync::Arc,
 };
 
+
```

Comment:
Blocker: Please revert this change before merging, it will cause `rustfmt` to fail.

As a longer-term fix, can you open a ticket to add the relevant workflow and docker files to the workflow path filters?